src/aoc2020/day1.clj

```diff
@@ -0,0 +1,143 @@
+(ns aoc2020.day1
+  (:require [aoc2020.util :refer [file->seq]]))
+
+
+;; part 1
+
+(defn strs->ints [strs]
+  "parsing strings to integers"
+  (let [str->int (fn [str]
+                   (Integer/parseInt str))]
+    (map str->int strs)))
+
+(defn int->compl [n sum]
```

Comment:
doesn't make sense to name this as a transform with the current naming. i.e., int is not domain specific naming, but compl is domain specific naming

Reply:
how about the name `complement`